hs/src/Reach/Connector/ALGO.hs

```diff
@@ -2967,8 +2980,13 @@ doWrapData tys mk = do
 
 cmeth :: Int -> CMeth -> App ()
 cmeth sigi = \case
-  CApi who sig _ which tys doWrap -> do
-    block_' (bunpack who) $ do
+  CAlias alias (CApi {..}) -> do
```

Comment:
Interesting... you're making the real thing available also AND the alias. That makes sense